test/sshkit_functional_test.exs

```diff
@@ -148,7 +148,8 @@ defmodule SSHKitFunctionalTest do
   end
 
   describe "upload/3" do
-    @tag boot: [@bootconf, @bootconf]
+    @describetag boot: [@bootconf, @bootconf]
```

Comment:
I added a `@describetag` here instead of annotating each individual test with `@tag`. For reference: https://hexdocs.pm/ex_unit/ExUnit.Case.html#module-module-and-describe-tags

Reply:
💥 **TIL** 🎉